### PR TITLE
Dump binding table in PSID format

### PR DESCRIPTION
**Pull request by dpino**
This PR depends on #160.

It's not necessary to do any change for configuration file dumping as it doesn't reference to PSID data.

### CODE REVIEW

File: src/apps/lwaftr/dump.lua [hunk 1]
```diff
@@ -36,6 +36,7 @@ function dump_configuration(lwstate)
    local etharr = set('aftr_mac_b4_side',  'aftr_mac_inet_side', 'b4_mac',  'inet_mac')
    local ipv4arr = set('aftr_ipv4_ip')
    local ipv6arr = set('aftr_ipv6_ip')
+   -- TODO: Skip compiled filters
```
Comment (aperezdc) [inline src/apps/lwaftr/dump.lua]: This `TODO` does not seem to be related to the rest of the patch (or maybe I am missing something here?)

**Comment by aperezdc** [comment]
LGTM.

File: src/apps/lwaftr/dump.lua (hunk 1 above)
Comment (dpino) [inline src/apps/lwaftr/dump.lua]: Compiled filters hasn't landed yet.

**Comment by dpino** [comment]
Closing this PR as #297 invalidates it.
